docs/UserGuide/9-Comparison/TSDB-Comparison.md

```diff
@@ -0,0 +1,400 @@
+
+* Compression: 
+  * IoTDB supports many encoding and compression for time series, like RLE, 2DIFF, Gorilla, etc.. and Snappy compression.
+  In IoTDB, you can choose which encoding method you want, according to the data distribution. For more info, see [here](http://iotdb.apache.org/UserGuide/Master/2-Concept/3-Encoding.html).
+  * InfluxDB also supports encoding and compression, but you can not define which encoding method you want.
+  It just depends on the data type. For more info, see [here](https://docs.influxdata.com/influxdb/v1.7/concepts/storage_engine/).
+  * OpenTSDB and KairosDB use HBase and Cassandra in backend, and have no special encoding for time series.
+  
+* MQTT protocol support:
+  
+  MQTT protocol is an international standard and widely known in industrial users. only IoTDB and InfluxDB support user using MQTT client to write data.  
+
+* Running on Edge-side Device:
+  
+  Nowdays, edge computing is more and more popular, which means the edge device has more powerful compution resources. 
+  Deploying a TSDB on the edge side is useful for managing data on the edge side and serve for edge computing. 
+  As OpenTSDB and KairosDB rely another DB, the architecture is a little heavy. Especially, it is hard to run Hadoop on the edge side.
+
+* Multi-instance Sync:
+  
+  Ok, now we have many TSDB instances on the edge-side. Then, how to upload their data to the data center, to form a ... data lake (or ocean, river,..., whatever).
+  One choice is read data from these instances and write the data point by point to the data center instance.
+  IoTDB provides another choice, just uploading the data file into the data center incrementally, then the data center can support service on the data. 
+  
+* JDBC driver:
+
+  Now only IoTDB supports a JDBC driver (though not all interfaces are implemented), and makes it possible to integrate many other JDBC drvier based softwares.
+
+* Standard SQL:
+
+  As mentioned, the integration of IoTDB and Calcite is alomost done (a PR has been submitted), which means IoTDB will support Standard SQL.
+  
+* Spark and Hive integration:
+
+  It is very very important that letting big data analysis software to access the data in database for more complex data analysis.
+  IoTDB supports Hive-connector and Spark connector for better integration. 
+
+*  Writing data to NFS (HDFS):
+  Sharing nothing architecture is good, but sometimes you have to add new servers even your CPU and memory is idle but the disk is full...
+  Besides, if we can save the data file directly to HDFS, it will be more easy to use Spark and other softwares to analyze data, without ETL.
+  
+  * IoTDB supports write data locally or on HDFS directly. IoTDB also allows user extend to store data on other NFS.
+  * InfluxDB, KairosDB have to write data locally.
+  * OpenTSDB has to write data on HDFS.
+    
+**Conclusion**:    
+  
+  We can find that IoTDB has many interesting features that other TSDBs do not support.
```

Comment:
I think "interesting" is not suitable here, which seems to be not necessary. If i am a user of iotdb, what i want to see in the conclusion is " IoTDB has more useful and practical function and features than other TSDBs".